docs/usage.rst

```diff
@@ -139,6 +139,24 @@ Command-Line Arguments
                This file is only created if the input type is "fmriprep" or "nibabies".
 
 
+**************
+Minimal Inputs
```

Comment:
Look much clearer! I'd further divide those into `/anat` and `/func` and add a generic file name that would correspond to the input like so:
- `/anat`:
   - native image (e.g.`sub-xxx_ses-y_desc-preproc_T1w.nii.gz`)
   - ...
- `func`:
   - preprocessed BOLD in std space (e.g., `sub-xxx_ses-y_task-zzz_run-i_space-MNI152NLin2009cAsym_res-2_desc-preproc_bold.nii.gz`)
   - ...
Might seem too explicit but I thought maximum pedagogy might pay off in the long term. Does it make sense?

Reply:
I took a crack at it. Would you mind taking a look at the updated version? https://xcp-d--1270.org.readthedocs.build/en/1270/usage.html#minimal-inputs